test/browser_tests/spec_suites/content/blog.js

```diff
@@ -134,4 +134,35 @@ describe( 'The Blog Page', function() {
   it( 'should include a page input with value set to 1', function() {
     expect( page.paginationPageInput.getAttribute( 'value' ) === 1 );
   } );
+
+  it( 'should set an initial height on filter expandable that does not cut off submit button when page loads with 5 filter options selected', function() {
+    // open filter and select 5 options
+    page.searchFilterShowBtn.click();
+    browser.sleep( 1000 );
```

Comment:
@contolini Yeah, I was trying to avoid all the callbacks, but that actually doesn't look as hellish as I expected. :)  I'll update to that & check out the alternative @sebworks suggested.